plans/testing-farm.fmf

```diff
@@ -37,7 +37,7 @@ discover:
         trap "cp -rv data/logs/ ${TMT_TEST_DATA}/" EXIT INT QUIT PIPE
 
         cd /root/kickstart-tests
-        scripts/run_travis.sh
+        scripts/run.sh
```

Comment:
We should use a more specific name. I'm afraid of people running this to run kickstart-tests.
Maybe `run-in-tmt` or `run-ci`?

Reply:
Good point.